### PR TITLE
add contributing.md

### PR DESCRIPTION
**Pull request by karuna24s**
Add CONTRIBUTING.md file. #4

### CODE REVIEW

**Comment by LNA** [comment]
LGTM 😀

**Comment by LNA** [comment]
Whoops! Can you add this to the guidelines ?
```git branch -D master
git fetch origin && git checkout -b name-of-new-feature-branch origin/master
git add patch
git commit -m "foo bar"
git push```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
```
Comment (karuna24s) [inline CONTRIBUTING.md]: @LNA This section mentions that we are using Github Flow (specifically Pull Requestions): Should I remove this section or create another section which is similar to this:
## Additional Guidelines to Follow Regarding Git Workflow after forking the repo:

```
git branch -D master
git fetch origin && git checkout -b name-of-new-feature-branch origin/master
git add patch
git commit -m "foo bar"
git push
```

Comment (LNA) [reply CONTRIBUTING.md]: I think it could just say ## Contributing

Comment (karuna24s) [reply CONTRIBUTING.md]: Sounds great I will add this heading.

**Comment by LNA** [comment]
Thanks! You can merge when you're ready.

**Pull request merged**